tests/api/api.spec.ts

```diff
@@ -556,6 +556,18 @@ describe("@dataform/api", () => {
           );
         });
     });
+    it("bigquery example with input backticks", () => {
+      return query
+        .compile("select 1 from `tada-analytics.df_integration_test.example_view`", {
```

Comment:
please add a test with text that contains a backslash, i think there is still a bug in the code for mishandling that case. (this is the exact same problem as you fixed in the lexer).

Reply:
Good point!